### PR TITLE
Update gun nation

### PR DESCRIPTION
**Pull request by duarte**
Updates the gun nation page to use the new branding, and link directly to Desert Fire instead of advertising it as releasing in October…

localhost-8000-immersive html 2

### CODE REVIEW

File: src/css/_onward.scss [hunk 1]
```diff
@@ -137,27 +144,48 @@
...
+    .trailer-button,
+    .trailer-duration,
     .section-title {
-      strong {
-        font-weight: 600;
+      display: none;
+    }
+    .watch-now {
+      display: inline-block;
+      color: #fb0;
```
Comment (akash1810) [inline src/css/_onward.scss]: can this be a variable?

File: src/css/_youtube.scss [hunk 2]
```diff
@@ -107,17 +107,16 @@
     text-shadow: 0 0 36px rgba(51, 51, 51, 0.4);
   }
   &--logo{
-    font-family: "Guardian Titlepiece Web";
-    font-size: 22px;
-    font-weight: 400;
+    width: 250px;
+    height: 121px;
+    overflow: hidden;
+    text-indent: -9999em;
```
Comment (akash1810) [inline src/css/_youtube.scss]: if this is really necessary, can we add a comment explaining why?

File: src/js/text/main.html [hunk 3]
```diff
@@ -59,16 +59,17 @@
 
   <div class='onward-signup'>
     <div class='onward-signup__container-title col'>
-      <div class='coming-soon'>Coming soon</div>
-      <div class='details'>Next up from the Guardian and Bertha’s groundbreaking documentary partnership</div>
+      <div class='coming-soon'>more documentaries</div>
```
Comment (akash1810) [inline src/js/text/main.html]: could we change the class name to reflect `more docs` too?

File: src/js/text/main.html [hunk 4]
```diff
@@ -59,16 +59,17 @@
...
     </div>
     <div class='onward-signup__contents col'>
       <div class='onward-signup__film-details col'>
         <span class='section-title'>Out on <strong data-sheet-attribute="nextDocDate"></strong></span>
         <span class='film-title' data-sheet-attribute="nextDocTitle"></span>
         <p class='film-synopsis' data-sheet-attribute="nextDocBody"></p>
-          <div class='docs--actions__trailer docs__shows-trailer' data-link-name="play-trailer">
-          <span class='docs--actions__trailer__duration'></span>
+        <a class='watch-now' href='https://www.theguardian.com/world/ng-interactive/2016/oct/14/desert-fire-the-world-cup-rebels-of-kurdistan-video?INTCMP=from_gun_nation_page'>Watch the documentary now</a>
+        <div class='docs--actions__trailer docs__shows-trailer trailer-button' data-link-name="play-trailer">
```
Comment (akash1810) [inline src/js/text/main.html]: from the screenshot, it doesn't look like there is a trailer on the page, is this still needed?

File: src/js/text/main.html [hunk 5]
```diff
@@ -59,16 +59,17 @@
...
+      <div class='details'>Watch more from the Guardian and Bertha’s groundbreaking documentary partnership</div>
     </div>
     <div class='onward-signup__contents col'>
       <div class='onward-signup__film-details col'>
         <span class='section-title'>Out on <strong data-sheet-attribute="nextDocDate"></strong></span>
         <span class='film-title' data-sheet-attribute="nextDocTitle"></span>
         <p class='film-synopsis' data-sheet-attribute="nextDocBody"></p>
-          <div class='docs--actions__trailer docs__shows-trailer' data-link-name="play-trailer">
-          <span class='docs--actions__trailer__duration'></span>
+        <a class='watch-now' href='https://www.theguardian.com/world/ng-interactive/2016/oct/14/desert-fire-the-world-cup-rebels-of-kurdistan-video?INTCMP=from_gun_nation_page'>Watch the documentary now</a>
```
Comment (akash1810) [inline src/js/text/main.html]: should we have a `data-link-name` attribute on this element for tracking?

**Comment by duarte** [comment]
@akash1810, can you please have a look at this? I think I've addressed everything, but not sure if we're better off removing the SEO text or keeping it — happy to update either way.

File: src/css/_youtube.scss (hunk 2 above)
Comment (akash1810) [inline src/css/_youtube.scss]: Worth checking with the seo team if there is a cleaner way to do this as this feels super hacky.

**Comment by duarte** [comment]
Removed @akash1810

**Comment by akash1810** [comment]
👍

**Pull request merged**